src/structs/dmatrix_macros.rs

```diff
@@ -1118,21 +1118,21 @@ macro_rules! small_dmat_from_impl(
 
             /// Builds a matrix filled with the components provided by a vector.
             /// The vector contains the matrix data in column-major order.
-            /// Note that `from_col_vec` is a lot faster than `from_row_vec` since a `$dmat` stores its data
+            /// Note that `from_col_vector` is a lot faster than `from_row_vector` since a `$dmatrix` stores its data
             /// in column-major order.
             ///
             /// The vector must have at least `nrows * ncols` elements.
             #[inline]
-            pub fn from_col_vec(nrows: usize, ncols: usize, vec: &[N]) -> $dmat<N> {
-                assert!(nrows * ncols == vec.len());
+            pub fn from_col_vector(nrows: usize, ncols: usize, vector: &[N]) -> $dmatrix<N> {
```

Comment:
Thanks for spotting this. It looks like I forgot to replace `col` by `column` almost everywhere!